tutorials/filter/src/main/java/com/datatorrent/tutorial/filter/Application.java

```diff
@@ -0,0 +1,44 @@
+package com.datatorrent.tutorial.filter;
+
+import org.apache.apex.malhar.lib.fs.FSRecordReaderModule;
+import org.apache.apex.malhar.lib.fs.GenericFileOutputOperator.StringFileOutputOperator;
+import org.apache.hadoop.conf.Configuration;
+
+import com.datatorrent.api.DAG;
+import com.datatorrent.api.StreamingApplication;
+import com.datatorrent.api.annotation.ApplicationAnnotation;
+import com.datatorrent.contrib.formatter.CsvFormatter;
+import com.datatorrent.contrib.parser.CsvParser;
+import com.datatorrent.lib.filter.FilterOperator;
+
+/**
+ * Simple application illustrating file input-output
+ */
+@ApplicationAnnotation(name="FilterExample")
+public class Application implements StreamingApplication
+{
+
+  @Override
+  public void populateDAG(DAG dag, Configuration conf)
+  {
+    
+    FSRecordReaderModule recordReader = dag.addModule("recordReader", FSRecordReaderModule.class);
+    CsvParser csvParser = dag.addOperator("csvParser", CsvParser.class);
```

Comment:
Test cases with data generator cannot compare correctness of output with the golden result set (expected output). It was based on the review comments one of the other PRs where reviewer mentioned that test should check for correctness of results rather than just checking if output file exists.